packages/itwin/grouping-mapping-widget/src/widget/components/SortableHorizontalTile.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from "react";
+import { useSortable } from "@dnd-kit/sortable";
+import type { HorizontalTileProps } from "./HorizontalTile";
+import { HorizontalTile } from "./HorizontalTile";
+import { CSS } from "@dnd-kit/utilities";
+import { SvgDragHandleVertical } from "@itwin/itwinui-icons-react";
+
+interface SortableHorizontalTileProps extends HorizontalTileProps {
+  id: string;
+}
+
+const SortableHorizontalTile = ({id, ...props}: SortableHorizontalTileProps) => {
+  const {
+    attributes,
+    listeners,
+    isDragging,
+    setNodeRef,
+    transform,
+    transition,
+  } = useSortable({id});
+
+  const style = {
+    transform: CSS.Transform.toString(transform),
+    transition,
+  };
+
+  useEffect(() => {
+    if (!isDragging) {
+      return;
+    }
+    document.body.style.cursor = "grabbing";
```

Comment:
This looks like a hack. What does this do?

Reply:
Globally change the cursor to grabbing when actively dragging around the tile.